boards.txt

```diff
@@ -711,6 +711,70 @@ adafruit_feather_m4.menu.debug.off=Off
 adafruit_feather_m4.menu.debug.on=On
 adafruit_feather_m4.menu.debug.on.build.flags.debug=-g
 
+# Adafruit Feather M4 CAN (SAME51)
+# ------------------------------
+adafruit_feather_m4_can.name=Adafruit Feather M4 CAN (SAMD51)
```

Comment:
change to (SAME51)

Reply:
Uhh... So does this mean green light for Controller Area Networking? Im already feeling the Buzz. 

Have been doing work on a controller using the SimpelFOC lib., integrating two full bridges for controlling 3phase, steppers, high curretn LED strings. Im talking 250w per lamp. Hortycultural. SAME51 support rocks!

https://github.com/Juanduino/THE-DIMMER_M4SDRV_V1 (its still in the mold).

Reply:
Fixed @ladyada

Reply:
Just a qustion regarding the CAN ports. Why not break out both CAN FD channels. CAN0 is on PortA22/23. Same as some of the TCC0. Anyways. since you already have those pins out. It could be just another choice. The cool thing about two channels is obvious. One main controller network and subnetworks.

Reply:
Oh its the same Sercom as USB. Is that a crash?